.eslintignore

```diff
@@ -5,3 +5,5 @@ migrations-*
 coverage
 node_modules
 assets
+dist
+tools/
```

Comment:
We've never really talked about a tools folder - is it worth adding to some doc somewhere?

Reply:
This is ignored because the `api-tests` have their own eslint config - I agree this isn't ideal, I think this goes under #2191 where they'll use the same config